demux/demux.c

```diff
@@ -3652,7 +3652,7 @@ static struct demux_cached_range *find_cache_seek_range(struct demux_internal *i
 
     for (int n = 0; n < in->num_ranges; n++) {
         struct demux_cached_range *r = in->ranges[n];
-        if (r->seek_start != MP_NOPTS_VALUE) {
+        if (r->seek_start != MP_NOPTS_VALUE || r->is_eof) {
```

Comment:
shouldn't this be `is_bof` to as a natural pair to seek_start?
I think `r->seek_start != MP_NOPTS_VALUE || r->is_bof` might actually be an invariant.

Reply:
Updated, I don't think it makes any practical difference.